packages/template-retail-react-app/app/contexts/index.js

```diff
@@ -40,65 +40,51 @@ export const CategoriesProvider = ({treeRoot = {}, children, locale}) => {
 
     const api = useCommerceAPI()
     const [root, setRoot] = useState({
-        // map over the server-provided cat
+        // Clone the server-provided category
         ...treeRoot[DEFAULT_ROOT_CATEGORY],
         [itemsKey]: treeRoot[DEFAULT_ROOT_CATEGORY]?.[itemsKey]?.map((item) => ({
             ...item
         }))
     })
 
     const fetchCategoryNode = async (id, levels = 1) => {
+        const storageKey = `${LOCAL_STORAGE_PREFIX}${id}-${locale}`
+        const storageItem = JSON.parse(window.localStorage.getItem(storageKey))
+
         // return early if there's one that is less than stale time
-        const storageItem = JSON.parse(
-            window.localStorage.getItem(`${LOCAL_STORAGE_PREFIX}${id}-${locale}`)
-        )
-        if (storageItem || Date.now() < storageItem?.fetchTime + CAT_MENU_STALE_TIME) {
+        if (storageItem && Date.now() < storageItem.fetchTime + CAT_MENU_STALE_TIME) {
```

Comment:
`||` meant that we would _always_ return the stored data, regardless of staleness.

Reply:
Gotcha, my fault on missing this, I missed the `||` which was the relevant bit